catalog/resource_connection.go

```diff
@@ -92,9 +99,6 @@ func ResourceConnection() *schema.Resource {
 			return common.StructToData(conn, s, d)
 		},
 		Update: func(ctx context.Context, d *schema.ResourceData, c *common.DatabricksClient) error {
-			if d.Get("owner") != "" {
```

Comment:
We would also need to update owner first and then rest of the fields + add rollback mechanism in Update right?

Reply:
fixed